website/community/technical-steering/_index.md

```diff
@@ -0,0 +1,54 @@
+
+The committee will schedule a Technical Steering meeting where the proposer presents the document. Committee members and interested contributors will challenge and discuss the idea. This meeting will be recorded.
+
+A decision on how to proceed with the document, including any recommended improvements, will be made within the same meeting. The decision and any required changes will be documented and shared.
+
+Only challenges requiring significant development effort or broad cross-team impact will be considered.
+
+### Why Do We Need It?
+Technical Steering ensures that major technical changes align with Gardener's architectural principles and long-term sustainability. It prevents fragmentation and promotes consistency in the project's evolution. The process helps ensure that major technical investments deliver value to both Gardener's users and maintainers.
+
+### Who is on The Technical Steering Committee?
+The committee is elected by the project's active committers and contributors and consists of a small, odd number of members (at least three) to ensure majority decision-making. Currently, the committee members are:
+- [Johannes Scheerer](https://github.com/scheererj)
+- [Rafael Franzke](https://github.com/rfranzke)
+- [Vedran Lerenc](https://github.com/vlerenc)
+
+### How Does It Work?
+#### 1. Document Submission
+- A proposer drafts a **technical design document** (e.g. an enhancement proposal, etc.) outlining the proposed solution, with a particular focus on APIs and interfaces and shares it with any member of the Technical Steering Committee.
+- The document is linked on this community page, and a **meeting date** is scheduled **1-4 weeks out** (at least one week for review and preparation but no more than four weeks to ensure timely decisions).
+
+#### 2. Participation
+- Those who wish to **actively challenge or discuss** the document can **request an invitation** from any committee member.
+- Others are encouraged to **watch the recording** instead of attending live.
+
+#### 3. Meeting Process
+- The proposer presents the technical document.
+- Attendees provide feedback, ask questions, and challenge the document.
+- The session is recorded for transparency.
+
+#### 4. Decision Process
+- A **go/no-go decision** is made **during the meeting**, along with recommended improvements if needed.
+- The final decision and any necessary changes are **documented on this community page**.
+
+### Key Considerations for Topics
+A topic qualifies for Technical Steering if it:
+- Requires **significant development effort** (≥6 person-months), OR
+- Has a **moderate to high impact** on other teams (e.g., landscape stability, operations).
+
+<hr/>
+
+## Planned Topics
+
+- **Persistent Volume Autoscaler** (no paper/date yet)
```

Comment:
You could link https://github.com/gardener/pvc-autoscaler and GEP-29 (https://github.com/gardener/gardener/pull/10690)

Reply:
Thanks.